### PR TITLE
Typo and grammar fixes

### PR DESCRIPTION
**Pull request by iTechsTR**
Fixed typos and grammar mistakes

### CODE REVIEW

**Comment by magento-engcom-team** [comment]
@iTechsTR thank you for contributing. Please accept Community Contributors team invitation [here](https://github.com/orgs/magento/invitation) to gain extended permissions for this repository.

**Comment by devops-devdocs** [comment]
An admin must run tests on this PR before it can be merged.

File: community/resources/support.md [hunk 1]
```diff
@@ -8,11 +8,11 @@ menu_order: 100
 menu_node: parent
 ---
 
-If you have issues, receive error messages, or need additional How Tos, see our [Magento Support Knowledge Base](https://support.magento.com). The knowledge base includes a growing number of troubleshooting, how to, and FAQ articles for Magento 2 Commerce, Commerce (Cloud), and Open Source. We are constantly adding new content for Magento platform, services, modules, and much more.
+If you have issues, receive error messages, or need additional How To's, see our [Magento Support Knowledge Base](https://support.magento.com). The knowledge base includes a growing amount of troubleshooting, how to, and FAQ articles for Magento 2 Commerce, Commerce (Cloud), and Open Source. We are constantly adding new content for Magento platform, services, modules, and much more.
```
Comment (hguthrie) [inline community/resources/support.md]: Good idea. How about a little more fixes...

```suggestion
If you have issues, receive error messages, or need additional How To content, see our [Magento Support Knowledge Base](https://support.magento.com). The knowledge base includes a growing collection of troubleshooting, how-to, and FAQ articles for the Magento 2 Commerce, Commerce Cloud, and Open Source products. We continually add new content for Magento platform, services, modules, and much more.
```

Comment (iTechsTR) [reply community/resources/support.md]: Perfect!

**Pull request merged**